quickstarts/botolytics/config.yml

```
@@ -0,0 +1,33 @@
+id:
```

```suggestion
id:
```
Please mention id here or remove